### PR TITLE
Task #627544 [TOKEN API][Swagger][2.2.1] Authentication Token by romsid

### PR DESCRIPTION
**Pull request by tupm-1335**
### Related Tickets
- [#627544](https://dev.sun-asterisk.com/issues/627544)

### Description
- [x] Implement swagger for API 2.2.1 Authentication Token by romsid

### Your checklist for this pull request
- [x] Have you self-test?
- [x] I had build successful before create PR.
- [ ] All existing and new tests are passing.
- [ ] I updated status my task
- [ ] Code has Unit Test?
- [x] Klint format is passed.
- [ ] Are you add new properties to `application.yml` and add sample to `/resources/application-*.example`?

### Configuration properties in application.yml
*Please describe key and value here*

### Notes
*(Other notes)*
xxx

### CODE REVIEW

File: src/main/kotlin/com/devh/vitstore/model/dto/AuthTokenByRomsidRequest.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import java.io.Serializable
+import javax.validation.constraints.NotBlank
+import javax.validation.constraints.Pattern
+
+data class AuthTokenByRomsidRequest(
+    val romsid: String,
+
+    @field:NotBlank
+    @field:Pattern(regexp = "REGISTRATION")
```
Comment (huorlk-0450) [inline src/main/kotlin/com/devh/vitstore/model/dto/AuthTokenByRomsidRequest.kt]: Chỗ này KH họ comment chỉ được duy nhất REGISTRATION hở e?

Comment (tupm-1335) [reply src/main/kotlin/com/devh/vitstore/model/dto/AuthTokenByRomsidRequest.kt]: @huorlk-0450 vưng anh ạ, trc e để là login thì KH nó cmt là REGISTRATION.

Comment (huorlk-0450) [reply src/main/kotlin/com/devh/vitstore/model/dto/AuthTokenByRomsidRequest.kt]: @tupm-1335 LOGIN là sai, 2.2.2 mới LOGIN
Theo ID API diagram với API change password nó gọi đến cái 2.2.1 này

Comment (tupm-1335) [reply src/main/kotlin/com/devh/vitstore/model/dto/AuthTokenByRomsidRequest.kt]: @huorlk-0450 thế e để cả REGISTRATION CẢ PWCHANGE nhé

Comment (huorlk-0450) [reply src/main/kotlin/com/devh/vitstore/model/dto/AuthTokenByRomsidRequest.kt]: ok

**Review by huorlk-0450 — APPROVED** [review]

**Pull request merged**